include/zephyr/input/input_analog_axis.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_H_
+#define ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_H_
+
+#include <stdint.h>
+#include <zephyr/device.h>
+
+/**
+ * @brief Keyboard Matrix API
+ * @defgroup input_kbd_matrix Keyboard Matrix API
+ * @ingroup io_interfaces
+ * @{
+ */
+
+/**
+ * @brief Analog axis calibration.
```

Comment:
it feels like this could use a detailed description in addition to the brief one, with a general description of in/out mapping and deadzone?

Reply:
Well there's more details in the field descriptions AND the binding, I'd rather avoid having even more duplication, in case this has to be updated, but it's worth adding some reference to how it's populated and what to use to change them I guess, added a paragraph with those.